src/gui/DatabaseManager.cpp

```diff
@@ -25,3 +25,23 @@ bool DatabaseManager::createNewProduct(int pId, QString pName, QString pCategori
 
     return false;
 }
+
+bool DatabaseManager::createNewProductInstance(int pId, int pInstId, QString pInstlocation, int pInstQuantity)
+{
+    bakcyl::common::ProductInstance newProductInstance;
+
+    newProductInstance.setProductId(std::uint64_t(pId));
+    newProductInstance.setId(std::uint32_t(pInstId));
+    newProductInstance.setLocationId(pInstlocation.toStdString());
+    newProductInstance.setQuantity(std::uint32_t(pInstQuantity));
+
+    bakcyl::core::Core::MethodResult queryResult = bakcyl::core::Core().createProductInstance(newProductInstance);
+
+    // TODO: Tell the user what happened
+    if(queryResult == bakcyl::core::Core::MethodResult::SUCCESS)
```

Comment:
return queryResult == bakcyl::core::Core::MethodResult::SUCCESS

Reply:
Teraz to wystarczy, ale w innych przypadkach do qml będzie przekazywana wiadomość odnośnie istoty błędu #46